src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -55,14 +31,25 @@
                                 </label>
                             </h5>
                             <div class="col-sm-8">
-                                <select class="form-control<c:if test="${empty fsForm.courses}"> text-color-red</c:if>"
-                                    name="<%= Const.ParamsNames.COURSE_ID %>"
-                                    id="<%= Const.ParamsNames.COURSE_ID %>">
-                                    <c:forEach items="${fsForm.coursesSelectField}" var="option">
-                                        <option <c:forEach items="${option.attributes}" var="attr"
-                                        > ${attr.key}="${attr.value}"</c:forEach> >${option.content}</option>
-                                    </c:forEach>
-                                </select>
+                                <c:choose>
+                                    <c:when test="${fsForm.courseIdEditable}">
+                                        <select class="form-control<c:if test="${fsForm.showNoCoursesMessage}"> text-color-red</c:if>"
+                                            name="<%= Const.ParamsNames.COURSE_ID %>"
+                                            id="<%= Const.ParamsNames.COURSE_ID %>">
+                                            <c:forEach items="${fsForm.coursesSelectField}" var="option">
+                                                <option <c:forEach items="${option.attributes}" var="attr"
+                                                > ${attr.key}="${attr.value}"</c:forEach> >${option.content}</option>
+                                            </c:forEach>
+                                        </select>
+                                    </c:when>
+                                    <c:otherwise>
+                                        <h5>
+                                            <div class="form-control-static">
+                                                    ${fsForm.courseIdForNewSession}
+                                            </div>
+                                        </h5>
```

Comment:
It's not actually valid, but this h5 inside a div thing is used several times on the form in instructorFeedbacks.jsp. I must have mistakenly thought that was also done here. 

I'll make an issue for other h5s after this has been merged.